contentcuration/contentcuration/management/commands/exportchannel.py

```diff
@@ -108,23 +108,23 @@ def queue_get_return_none_when_empty():
 
 
     # kolibri_license = kolibrimodels.License.objects.get(license_name=license.license_name)
-
-    with ccmodels.ContentNode.objects.delay_mptt_updates():
-        for node in iter(queue_get_return_none_when_empty, None):
-            logging.debug("Mapping node with id {id}".format(
-                id=node.pk))
-
-            children = (node.children.
-                        # select_related('parent', 'files__preset', 'files__file_format').
-                        all())
-            node_queue.extend(children)
-
-            kolibrinode = create_bare_contentnode(node)
-
-            if node.kind.kind == content_kinds.EXERCISE:
-                create_perseus_exercise(node)
-            if node.kind.kind != content_kinds.TOPIC:
-                create_associated_file_objects(kolibrinode, node)
+    with transaction.atomic():
```

Comment:
Yeah, it's both faster, and won't have inconsistent intermediate database states.